### DISCUSSION TITLE
Problem When Trying To Add Cheat Sheets

### DISCUSSION THREAD

**Discussion by StylaxGaming** [SnosMe/awakened-poe-trade#867] [Q&A]
When i try to add a cheat sheet to the macro it doesnt allow me to do so, am i oing something wrong, is there a specific size of image i have to use?

**Comment by kissmyastroturf** [comment]
I am also experiencing this issue.  It started with one of the recent updates.  It seems that the overlay will only let you add JPG images now where before it would let you add other image types, like PNG.  My use case is using the Windows Snipping Tool to capture a selection, then saving it to an image.  The default image format for the Windows Snipping Tool is PNG.

**Comment by SnosMe** [comment]
There is currently limit of 800KB.
As you said it was introduced in the latest release (unintentionally), and will be fixed in the next one.
